rules/CodingStyle/Rector/Assign/ManualJsonStringToJsonEncodeArrayRector.php

```diff
@@ -294,4 +297,17 @@ private function getNextExpression(Node $node): ?Node
 
         return $currentExpression->getAttribute(AttributeKey::NEXT_NODE);
     }
+
+    private function createJsonEncodeAssign(Expr $assignExpr, Array_ $jsonArray): Assign
+    {
+        if ($this->reflectionProvider->hasClass('Nette\Utils\Json')) {
+            return $this->jsonEncodeStaticCallFactory->createFromArray($assignExpr, $jsonArray);
+        }
+
+        $jsonDataAssign = new Assign($assignExpr, $jsonArray);
+        $jsonDataVariable = new Variable('jsonData');
+        $jsonDataAssign->expr = $this->nodeFactory->createFuncCall('json_encode', [$jsonDataVariable]);
```

Comment:
To mimic nette-json more closely, should this use e.g. `JSON_THROW_ON_ERROR` etc ?

see https://github.com/nette/utils/blob/773238807e7aaa9fe354908a32955076972af735/src/Utils/Json.php#L36 which flags nette uses by default

Reply:
I think this is not to mimic nette-json, but original behavior.
The added value in `Nette\Utils\Strings` (including all the error handling) is just a bonus.